### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by maasencioh**
Hi @targos, could you also help me to review the code in master?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,19 @@ __options__
 
 ## Example
 
-See the file [test/test-main.js](https://github.com/image-js/hog/test), which takes the path of an image and return the HOG descriptor as a vector.
+```js
+'use strict';
 
+const {default: Image} = require('image-js');
```
Comment (targos) [inline README.md]: `const {Image} = require('image-js')` should work

File: src/index.js [hunk 2]
```diff
@@ -1,177 +1,177 @@
...
-  for (var y = 0; y < blocksHigh; y += blockStride) {
-    for (var x = 0; x < blocksWide; x += blockStride) {
-      var block = getBlock(histograms, x, y, blockSize);
-      normalize(block, norm);
-      blocks.push(block);
+    const {
+        blockSize = 2,
+        norm = 'L2'
+    } = options;
+    const blockStride = options.blockStride || (blockSize / 2);
```
Comment (targos) [inline src/index.js]: you can put that in the destructuring: 
```js
const {
    blockSize = 2,
    blockStride = blockSize / 2
    norm = 'L2'
} = options;
```

File: src/index.js [hunk 3]
```diff
@@ -1,177 +1,177 @@
...
+    for (var y = 0; y < blocksHigh; y += blockStride) {
+        for (var x = 0; x < blocksWide; x += blockStride) {
+            var block = getBlock(histograms, x, y, blockSize);
+            normalize(block, norm);
+            blocks.push(block);
+        }
     }
-  }
-  return Array.prototype.concat.apply([], blocks);
+    return Array.prototype.concat.apply([], blocks);
```
Comment (targos) [inline src/index.js:44]: why not just return blocks?

File: src/index.js (hunk 2 above)
Comment (maasencioh) [reply src/index.js]: I wasn't sure, thanks

File: README.md (hunk 1 above)
Comment (jajoe) [reply README.md]: Oki, we change that

File: src/index.js (hunk 3 above)
Comment (jajoe) [reply src/index.js:44]: not the good shape, it doesn't work without Array.prototype.concat.apply([], blocks).

File: src/index.js (hunk 2 above)
Comment (jajoe) [reply src/index.js]: oki we'll do that, we were not sure that it works.

**Comment by jajoe** [comment]
I'm waiting for you permission to merge pull request

File: src/index.js (hunk 2 above)
Comment (targos) [reply src/index.js]: The best way to be sure is to add a test that triggers this code path ;)

**Review by targos — APPROVED** [review]

**Pull request merged**